packages/notte-integrations/src/notte_integrations/pdf/pdf_options.py

```diff
@@ -0,0 +1,26 @@
+import os
+from enum import StrEnum
+
+from notte_integrations.pdf.docling.config import DEFAULT_CONFIG
+from notte_integrations.pdf.docling.serve import DoclingPDFHandler
+
+
+class PDFOptionEnum(StrEnum):
+    DOCLING = "Docling"
+
+
+class PDFChoiceHandler:
+    @classmethod
+    def create_from_env(cls):
```

Comment:
Is the url going to change ? Does it really have to be set in the env file ? I would remove `create_from_env` and just put that in the constructor of the class